src/RNCamera.js

```diff
@@ -102,6 +102,12 @@ type StateType = {
 
 type Status = 'READY' | 'PENDING_AUTHORIZATION' | 'NOT_AUTHORIZED';
 
+const CameraStatus = {
```

Comment:
export this so end user can use it as well

Reply:
Is is exported on `RNCamera.Constants`